MekHQ/src/mekhq/gui/dialog/MarkdownEditorDialog.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * TextAreaDialog.java
```

Comment:
Needs MekHQ header, meaningful comment for the class, preferably meaningful javadoc comments for the public methods as well.

Reply:
Added javadocs in 0a3caacf0.